### DISCUSSION TITLE
Common Fields across Projects

### DISCUSSION THREAD

**Discussion by jason-ola** [community/community#8319] [Projects and Issues]
Hi all,

my team are experimenting Github Project for a small number of team. in our case PM are the role to create feature ISSUEs from PM Project and pass over (issue with multi-project) requirements to other team owning another ENG Project. 

I realized Custom Fields defined in PM Project can't be shared with other ENG Projects like other predefined fields like repo, assignee, but we do have the need to structure the ISSUE information using a company wise Common Set of Custom Fields that's visible to other projects.

Is there a way to achieve Common Fields?

thanks,

**Comment by Ormstrup** [comment]
This feature is very sought for from our perspective as well. We currently use the specific issues in multiple projects, as there are different departments that work on the specific issues, within their own project. At lot of info current needs to be put in manually in order to keep the overview.

The main thing here would be to have more fields connected to the specific issue, which then can be transferred to the different projects.

**Comment by volkantash** [comment]
Biz de bekliyoruz.
(We are waiting too)

**Comment by jimmycarr** [comment]
This feature is a bit of a dealbreaker for us right now. We have issues sitting on multiple boards with fields we want to be updated across all of them (like estimates for example). Really hope this gets built into the product soon.

**Comment by acolchagoff** [comment]
They appear to be exploring this feature, see: https://github.com/orgs/community/discussions/10853
